google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -825,7 +827,7 @@ public OperationFuture<ResponseT, MetadataT> call() {
             isRetry = true;
 
             if (operationName == null) {
-              GrpcCallContext context = newCallContext(null, instanceName, initialRequest, method);
+              GrpcCallContext context = newAdminCallContext(instanceName, initialRequest, method);
```

Comment:
This isn't directly related, but a small cleanup to prevent each of these methods to invoke `newCallContext(null, ...)`, and instead use a more descriptive method.